Makefile

```diff
@@ -192,7 +192,7 @@ $(WEB_INSTALL_GUARD): webui/package.json
 	cd webui && npm install
 
 export GENERATE_SOURCEMAP := false
-$(WEB_BUILD_FILES): $(WEB_SRC_FILES) $(WEB_INSTALL_GUARD)
+${WEB_BUILD_FILES} &: $(WEB_SRC_FILES) $(WEB_INSTALL_GUARD)
```

Comment:
The &: was a good find!

Reply:
Yeah, I thought `make` did this by default, but it turns out it doesn't! TIL!